### PR TITLE
copy readme to dist/proj folder

### PR DESCRIPTION
**Pull request by maxisam**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,6 +48,7 @@
     "karma-jasmine-html-reporter": "^0.2.2",
     "protractor": "~5.3.0",
     "ts-node": "~5.0.1",
-    "tslint": "~5.9.1"
+    "tslint": "~5.9.1",
+    "cpx": "^1.5.0"
```
Comment (muratcorlu) [inline package.json]: Why just `cp` is not enough?

File: package.json [hunk 2]
```diff
@@ -4,11 +4,12 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "build": "ng build ngx-script-loader --prod",
+    "build": "ng build ngx-script-loader --prod && npm run copy-readme",
```
Comment (muratcorlu) [inline package.json]: Maybe that should be a part of publish process, not build process 🤔

Comment (maxisam) [reply package.json]: I am fine either way. :)

**Comment by muratcorlu** [comment]
I'll make small changes. Thank you

**Pull request merged**